freqtrade/rpc/telegram.py

```diff
@@ -32,7 +32,7 @@ def init(config: dict) -> None:
     global _updater
 
     _CONF.update(config)
-    if not _CONF['telegram']['enabled']:
+    if not _CONF['telegram']['enabled'] or misc.args.backtesting:
```

Comment:
deactivate telegram via config in a earlier state if backtesting is enabled. So that this module doesn't have to be aware of it